src/Network/Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -183,13 +183,22 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
         public Hashtable Tag { get; set; }
 
         [Parameter(
+            ParameterSetName = "IdentityByUserAssignedIdentityId",
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "ResourceId of the user assigned identity to be assigned to Application Gateway.")]
         [ValidateNotNullOrEmpty]
         [Alias("UserAssignedIdentity")]
         public string UserAssignedIdentityId { get; set; }
```

Comment:
@akshaysngupta since the intention is to remove this cmdlet, you will want to add our [custom breaking change attribute](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#a-parameter-is-being-deprecated) on this parameter so users are notified of the change whenever they use it.

You should also add an entry to the [`upcoming-breaking-changes.md`](https://github.com/Azure/azure-powershell/blob/master/src/Network/documentation/upcoming-breaking-changes.md) file in the Network folder to call out this change (you can remove the content already in the file)